Cesium3DTilesSelection/src/Tile.cpp

```diff
@@ -846,7 +846,6 @@ void Tile::update(
     }
   }
 
-  // TODO: if there's no model, we can actually free any existing overlays.
   if (this->getState() == LoadState::Done &&
       this->getTileset()->supportsRasterOverlays() && this->getContent() &&
       this->getContent()->model) {
```

Comment:
Not directly related to this PR, but "supportsRasterOverlays" doesn't need to exist anymore right? Removing that would make this and the "else if" logic a little clearer to read. Opened this issue for it:  #489